code/workspaces/web-app/src/components/notebooks/NotebookSharingOptions.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  select: {
+    whiteSpace: 'normal',
+    maxWidth: 500,
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  title: {
+    fontWeight: 500,
+  },
+  description: {
+    marginTop: theme.spacing(1),
+  },
```

Comment:
I'm going to be really annoying but now that the size of text for the description and warning have changed, this margin is no longer necessary. Sorry!

In the mock-ups this text also uses a light grey colour that can be accessed with `theme.typography.colorLight`. Don't know whether it is worth updating this to match?